<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorldDataBankGrabber</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Select topic:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="TopicCmbBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select indicator:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="IndicatorCmbBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Select countries:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListView" name="CountiesLstView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Select dates:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>from:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="FromDateEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>1960</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1960</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>till:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="TillDateEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="date">
            <date>
             <year>1960</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="minimumDate">
            <date>
             <year>1960</year>
             <month>1</month>
             <day>1</day>
            </date>
           </property>
           <property name="displayFormat">
            <string>yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Path save:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PathSaveBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Select folder</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="GetCSVBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Get CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="GetXMLBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Get XML</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UpdateBtn">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>TopicCmbBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeIndicators(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UpdateBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PathSaveBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GetXMLBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>downloadXML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GetCSVBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>downloadCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeIndicators(int)</slot>
  <slot>updateConfiguration()</slot>
  <slot>selectFolder()</slot>
  <slot>downloadXML()</slot>
  <slot>downloadCSV()</slot>
 </slots>
</ui>
